### ISSUE TITLE
[Feature Request] Toggle between single image or 2x2

### ISSUE THREAD

**Issue by loeckchendasboesschen** [reuterbal/photobooth#158]
Hello everybody,

a feature of choosing between a single picture and the 2x2 grid would be really great. I'm just assuming that some people don't wanna pose for four pictures and would prefer a single one.
Would be great seeing this in the future.

**Comment by themaluxis** [comment]
Agreed with this one, could be great ! :) (ex: https://github.com/werdeil/pibooth)
